### ISSUE TITLE
All CAPD pipelines are failing because the new calico 3.22.1 images stop shipping cni_plugins

### ISSUE THREAD

**Issue by 12345lcr** [vmware-tanzu/tanzu-framework#2263]
**Bug description**
After calico images bumped to 3.22.1, the pr https://github.com/vmware-tanzu/tanzu-framework/pull/2229 to update the calico provider shows the CAPD and providers tests are failing.

This is because in newest calico images, we made it stop copying the irrelevant cni plugins like `bandwidth, flannel, host-local, loopback, portmap, tuning` into the nodes. These plugins should be retrieved from `cni_plugins` in bom instead.

But during CAPD testing that using calico as cni in management cluster, calico is installed by ClusterResourceSet first. And the management cluster creation will be stuck and all pods are hold in `ContainerCreating` state because `failed to find plugin "bandwidth" in path [/opt/cni/bin]` .

The sequence of installing cni_plugins and calico ClusterResourceSet should be adjusted.
**Affected product area (please put an X in all that apply)**

- [ ] APIs
- [x] Addons
- [ ] CLI
- [ ] Docs
- [ ] IAM
- [x] Installation
- [ ] Plugin
- [ ] Security
- [x] Test and Release
- [ ] User Experience

**Expected behavior**
The cni_plugins like `bandwidth, flannel, host-local, loopback, portmap, tuning` should be already exists in nodes before calico is deployed by ClusterResourceSet.

**Steps to reproduce the bug**
All pr pipelines are failing on this by far.
Locally run `make e2e-tkgctl-docker` could reproduce this problem

**Version** (include the SHA if the version is not obvious)
tkg 1.6.0
**Environment where the bug was observed (cloud, OS, etc)**
CAPD
**Relevant Debug Output (Logs, manifests, etc)**
```
Warning  FailedCreatePodSandBox  7m2s                   kubelet            Failed to create pod sandbox: rpc error: code = Unknown desc = failed to setup network for sandbox "5a7f7451a5a7472bb6f1bfd4838b7139f82cd20b54e2b7617d5e6bdf96ca3ab4": failed to find plugin "bandwidth" in path [/opt/cni/bin]
```

**Comment by 12345lcr** [comment]
I am reverting the calico 3.22.1 bump in 1.6.0 release

**Comment by 12345lcr** [comment]
Fixed by calico 3.22.1 bump. I am closing this.
